assets/device/classes/light.json

```diff
@@ -18,7 +21,10 @@
     "sv": "Använd den här enhetsklassen för lampor, oftast tillsammans med funktionerna `av/på`, `dim` och `lys upp_*`.",
     "no": "Bruk denne enhetsklassen for lamper, ofte sammen med funksjonene `av/på`, `demp` och `lys_*`.",
     "es": "Utiliza esta clase de dispositivo para las luces, generalmente junto con las capacidades de 'encendido/apagado', 'intensidad de la luz', y 'luz_*'.",
-    "da": "Brug denne enhedsklasse til lys, normalt sammen med `tændtslukket`, `dæmp` og `lys_*` funktionerne."
+    "da": "Brug denne enhedsklasse til lys, normalt sammen med `tændtslukket`, `dæmp` og `lys_*` funktionerne.",
```

Comment:
```suggestion
    "da": "Brug denne enhedsklasse til lys, normalt sammen med `onoff`, `dim` og `light_*` funktionerne.",
```

Reply:
This should also be fixed for the other languages in this description

Reply:
I see there is a Dutch translation missing here... Even though there is a dutch translation present in "title". Is there a reason as to why it's not included? Same goes for other device classes after a quick scan.

Reply:
Not sure where these descriptions are used exactly. I think for now it is fine to omit the Dutch translations.